jetty-http/src/main/java/org/eclipse/jetty/http/HttpCookie.java

```diff
@@ -22,13 +22,41 @@
 import java.util.concurrent.TimeUnit;
 
 import org.eclipse.jetty.util.QuotedStringTokenizer;
+import org.eclipse.jetty.util.StringUtil;
 
 // TODO consider replacing this with java.net.HttpCookie
 public class HttpCookie
 {
     private static final String __COOKIE_DELIM = "\",;\\ \t";
     private static final String __01Jan1970_COOKIE = DateGenerator.formatCookieDate(0).trim();
 
+    /**
+     *If this string is found within the comment parsed with {@link #isHttpOnlyInComment(String)} the check will return true
+     **/
+    private static final String HTTP_ONLY_COMMENT = "__HTTP_ONLY__";
+    /**
+     *These strings are used by {@link #getSameSiteFromComment(String)} to check for a SameSite specifier in the comment
+     **/
+    private static final String SAME_SITE_COMMENT = "__SAME_SITE_";
+    private static final String SAME_SITE_NONE_COMMENT = SAME_SITE_COMMENT + "NONE__";
+    private static final String SAME_SITE_LAX_COMMENT = SAME_SITE_COMMENT + "LAX__";
+    private static final String SAME_SITE_STRICT_COMMENT = SAME_SITE_COMMENT + "STRICT__";
+
+    public enum SameSite
+    {
+        NONE("None"), STRICT("Strict"), LAX("Lax");
+
+        private String attributeValue;
+        SameSite(String attributeValue) {
+            this.attributeValue = attributeValue;
+        }
+
+        @Override
+        public String toString(){
```

Comment:
Brace on newline.
Also, I don't like that `toString()` is used as a getter for `attributeValue`. I would prefer to have an explicit getter so that if in future we change `toString()` to include some other debugging string (e.g. `getClass().getSimpleName()` or `hashCode()`), we don't break the code.